app/views/cclow/home/_latest_additions.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="slider-container">
+  <div class="slider" data-slick='{"slidesToShow": 3, "slidesToScroll": 3}'>
+    <% @latest_additions.each do |item| %>
+      <div class="item-container">
+        <div class="item">
+          <div class="additions-kind"><%= item[:kind] %></div>
+          <div class="title"><a href="#"><%= item[:title] %></a></div>
+          <div class="meta">
+            <img class="flag" src=<%= asset_path "flags/#{item[:iso]}.svg"%> alt="">
+            <div><%= item[:jurisdiction] %>  |  <%= item[:addition_type] %>  |  <%= item[:date_passed] %></div>
```

Comment:
Maybe it's because of the data that we have but at the moment because we don't have some of these attributes, sometimes we are displaying unnecessary `|` . So we should check for the presence of the data and then join with pipes.

Reply:
could be like `item.values_at(:jurisdiction, :addition_type, :date_passed).compact.join(' | ').html_safe`

Reply:
Jurisdiction should be a link actually